.github/workflows/npm-test.yml

```diff
@@ -23,11 +23,12 @@ jobs:
           - ubuntu-latest
           - windows-latest
         node-version:
-          - '18.0'
+          - 18.0
```

Comment:
The quotes are actually necessary. Removing them causes this to be interpreted as Node 18, which causes the latest version of 18 to be used instead of 18.0.

Reply:
Sorry for that; I reverted the change to single quotes.